setup.cfg

```diff
@@ -177,6 +178,13 @@ exclude =
 rst-roles =
     # TODO look into these
     attr
+    cite
+    cite:ct
+    cite:cts
+    cite:p
+    cite:ps
+    cite:t
+    cite:ts
```

Comment:
It's pretty annoying to have to add all of these here, but I don't know of another way to automate this.

- [ ] Should briefly refer to this in the documentation guide since it took me a while to figure out how to deal with this.